artemis-console-extension/artemis-extension/packages/artemis-console-plugin/src/addresses/AddressesTable.tsx

```diff
@@ -43,7 +43,25 @@ export const AddressesTable: React.FunctionComponent<Navigate> = (navigate) => {
     { id: 'id', name: 'ID', visible: false, sortable: true, filterable: true },
     { id: 'name', name: 'Name', visible: true, sortable: true, filterable: true },
     { id: 'routingTypes', name: 'Routing Types', visible: true, sortable: true, filterable: true },
-    { id: 'queueCount', name: 'Queue Count', visible: true, sortable: true, filterable: true, filter: getQueueFilter, filterTab: 6 }
+    { id: 'queueCount', name: 'Queue Count', visible: true, sortable: true, filterable: true, filter: getQueueFilter, filterTab: 6 },
+    { id: 'internal', name: 'Internal', visible: false, sortable: true, filterable: true },
+    { id: 'temporary', name: 'Temporary', visible: false, sortable: true, filterable: true },
+    { id: 'autoCreated', name: 'Auto Created', visible: false, sortable: true, filterable: true },
+    { id: 'paused', name: 'Paused', visible: false, sortable: true, filterable: true },
+    { id: 'currentDuplicateIdCacheSize', name: 'Current Duplicate ID Cache Size', visible: false, sortable: true, filterable: true },
+    { id: 'retroactiveResource', name: 'Retroactive Resource', visible: false, sortable: true, filterable: true },
+    { id: 'unroutedMessageCount', name: 'Unrouted Message Count', visible: false, sortable: true, filterable: true },
+    { id: 'routedMessageCount', name: 'Routed Message Count', visible: false, sortable: true, filterable: true },
+    { id: 'MessageCount', name: 'Message Count', visible: true, sortable: true, filterable: true },
```

Comment:
The case of `messageCount` seems wrong. Can it be fixed in activemq-artemis `AddressField`?

Reply:
I fixed this in the broker via ARTEMIS-5518 and updated this PR.